src/test/suite/generateStructure.test.ts

```diff
@@ -0,0 +1,44 @@
+import { generateProjectStructure } from '../../utils/generateStructure.utils'
+import { describe, it, beforeEach, afterEach } from 'mocha'
+import * as fs from 'fs'
+import * as sinon from 'sinon'
+import { expect } from 'chai'
+
+describe('generateProjectStructure', () => {
+  let readdirSyncStub: sinon.SinonStub
+
+  beforeEach(() => {
```

Comment:
Mock the folder structure of an user in the mocha setup function like in the qssociation.test.ts file